src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -76,6 +93,18 @@
                 :message (str "Invalid level: " level)}))
     :info))
 
+(defn get-service-status-version
+  "Given a params map from a request, get out the service status version and
+   check whether it is valid. If not, throw an error."
+  [params]
+  (when-let [level (params :service-status-version)]
+    (try
+      (Integer. level)
+      (catch NumberFormatException e
```

Comment:
Given that trapperkeeper itself has a dependency on KS, won't we always have a KS dependency? Or are you saying we don't want to have a dependency on a specific version of KS if we don't have to?